src/app/crypto/services/utils.ts

```diff
@@ -15,27 +16,33 @@ async function passToHash(passObject: PassObjectInterface): Promise<{ salt: stri
   let salt;
   let hash;
 
-  // Those parameters are the same as the default parameters of node-argon2
-  // We explicitly pass them to hash to avoid compatibility ptoblems if node-argon2 changes them in the future
-  const params = {
-    hashLength: 32,
-    timeCost: 3,
-    memoryCost: 1 << 16,
-    parallelism: 4,
-    type: argon2.argon2id,
-    version: 0x13,
-  };
   if (passObject.salt) {
-    // salt is given, hence it's a double hash computed as argon2id(PBKDF2(pwd))
-    // Argon2id samples salt internally and stores it as part of output
+    // if salt =>   argon2id(PBKDF2(pwd))
     salt = CryptoJS.enc.Hex.parse(passObject.salt);
     const oldhash = CryptoJS.PBKDF2(passObject.password, salt, { keySize: 256 / 32, iterations: 10000 });
-    hash = await argon2.hash(oldhash, params);
+    const argonSalt = crypto.randomBytes(16);
+    hash = await argon2id({
```

Comment:
@sg-gs The library with blake3 had argon2, so I switched to it. 

Parameters are from OWASP recomendations: m=19456 (19 MiB), t=2, p=1 (Do not use with Argon2i) https://cheatsheetseries.owasp.org/cheatsheets/Password_Storage_Cheat_Sheet.html#argon2id